lxd/instance/drivers/driver_lxc.go

```diff
@@ -1979,6 +1979,11 @@ func (d *lxc) startCommon() (string, []func() error, error) {
 		return "", nil, errors.Wrap(err, "Load go-lxc struct")
 	}
 
+	// Ensure cgroup v1 configuration is set appropriately with the image using systemd
+	if d.localConfig["image.requirements.cgroup"] == "v1" && !shared.PathExists("/sys/fs/cgroup/systemd") {
+		return "", nil, errors.New("Failed to start container: requires host to use cgroups v1")
```

Comment:
We prefer `fmt.Errorf("Failed to start container: requires host to use cgroups v1")` for this